apps/rule-manager/client/src/ts/components/RuleForm.tsx

```diff
@@ -148,6 +149,32 @@ export const RuleForm = ({ruleId, onClose, onUpdate}: {
         })
     }
 
+    const unarchiveRuleHandler = () => {
+        if (!ruleFormData?.id || ruleState !== 'archived') {
+            return;
+        }
+
+        unarchiveRule(ruleFormData.id).then(data => {
+            if (data.status === 'ok'){
+                setRuleFormData(baseForm);
+                onClose();
```

Comment:
Do we really want to close the form on unarchive or archive? My feeling is that only clicking 'close' should close the sidebar, but you may feel differently!